JobConfig/primary/RMCGammaEndpointWriter.fcl

```diff
@@ -0,0 +1,11 @@
+#
+# RMC External photons, kmax = 101.9 MeV, saving events with a photon conversion
+#
+
+#include "Production/JobConfig/primary/RMCGammaWriter.fcl"
+
+physics.producers.generate.decayProducts.spectrum.ehi      : 101.9
```

Comment:
These numbers also appear in RMCExternalEndpoint, can those scripts be nested to avoid the duplication?

Reply:
I moved the measured and kinematic limit endpoints to the prolog and then reference them here, so there's now only one maintenance point if we want to update the values. Does this seem like a better setup?